courses/free-courses-ur.md

```diff
@@ -12,3 +13,9 @@
 ### HTML and CSS
 
 * [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
+
+
+### Javascript
+
+* [Javascript Tutorials In Urdu and Hindi](https://www.youtube.com/watch?v=Lgxgm-T9cgA&list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf) - Yahoo Baba
+* [Learn JavaScript in Urdu & Hindi - For Beginners - Introduction](https://www.youtube.com/watch?v=NKgdmmiPVUc&list=PL52B8F99BC6AD1647) - WP Academy
```

Comment:
Are you referring to this link: https://www.youtube.com/playlist?list=PL52B8F99BC6AD1647

If so, use the original title from the source and avoid creating new ones.

Reply:
Ok Man!